packages/server/tests/localAcceptance.env

```diff
@@ -15,3 +15,4 @@ HAPI_CLIENT_DURATION_RESET=3600000
 HAPI_CLIENT_ERROR_RESET=[50]
 FILTER_API_ENABLED=true
 ETH_POPULATE_SYNTHETIC_CONTRACT_RESULTS=true
+DEBUG_API_ENABLED=false
```

Comment:
Wouldn't we want this true, especially for acceptance test?

Reply:
@Nana-EC  well, since the acceptance tests are not yet merged with this PR I thought at first we would want this as false
Should I set it to true here

Reply:
Yeah, set it to true for local flows.
Also theres a bad merge in here on lines 18, 20 and 22